### ISSUE TITLE
pdf2htmlEX extensions

### ISSUE THREAD

**Issue by btbonval** [FinalsClub/karmaworld#434]
Do we still need pdf2htmlEX?

If so, do we still need those two pdf2htmlEX extensions which patch the upstream source?

Presently the documentation assumes we still need it and the patching is still required. Changes were made that might invalidate one or both assumptions.

**Comment by AndrewMagliozzi** [comment]
Yes we still need this. Not sure about the two extensions, however.

**Comment by btbonval** [comment]
I believe the extensions were exposed for the PDF viewer with the zoom buttons and chapter selection. I think @yourcelf made a universal zoom with CSS which worked even on the PDF pages, and I'm not sure if we support chapters now (we were going to support global chapters somehow).

I'll have to look at the code to know for sure. If it happened as I remembered, we no longer need a patched version of pdf2htmlEX, which will streamline the bootstrap process ever so slightly.

**Comment by AndrewMagliozzi** [comment]
cool
